sentry_sdk/integrations/sanic.py

```diff
@@ -276,16 +274,14 @@ async def sentry_wrapped_error_handler(request, exception):
 
 async def _legacy_handle_request(self, request, *args, **kwargs):
     # type: (Any, Request, *Any, **Any) -> Any
-    hub = Hub.current
-    if hub.get_integration(SanicIntegration) is None:
+    if sentry_sdk.get_client().get_integration(SanicIntegration) is None:
         return old_handle_request(self, request, *args, **kwargs)
```

Comment:
Just commenting, no changes requested here:
I guess we cannot use the `ensure_integration_handled_async` here, because that decorator would await on the `old_handle_request`

Reply:
Yep.